l10n/de_DE/LC_MESSAGES/civioffice.po

```diff
@@ -456,10 +598,281 @@ msgstr ""
+"Zugriff zu synchronisieren. (Es kann nur jeweils ein unoconv Prozess zur "
+"gleichen Zeit laufen.) Die Datei wird automatisch am unten angezeigten Ort "
+"erstellt. Bitte beachten: Der Pfad muss auf allen CiviCRM-Umgebungen auf "
+"diesem Server gleich sein. Anderenfalls ist das Sperren nur für diese "
+"CiviCRM-Instanz aktiv."
+
+#: templates/CRM/Civioffice/Form/DocumentRenderer/Settings/LocalUnoconv.tpl
+msgid ""
+"Run the DOCX document through the converter prior to processing in order to "
+"optimise/repair possibly corrupted CiviCRM tokens in the document."
+msgstr ""
+"DOCX-Dokumente vor der Verarbeitung durch den Konverter laufen lassen, um "
+"sie zu optimieren und mögliche korrupte CiviCRM-Tokens zu reparieren."
+
+#: templates/CRM/Civioffice/Form/DocumentRenderer/Settings/LocalUnoconv.tpl
+msgid ""
+"Whether to replace tokens using a PHPWord template processor, so that HTML "
+"in Live Snippets can be converted to OOXML."
+msgstr ""
+"Ob Tokens mit dem PHPWord-Vorlagen-Prozessor ersetzt werden, sodass HTML in "
+"Live-Snippets zu OOXML konvertiert werden kann."
+
+#: templates/CRM/Civioffice/Form/DocumentRenderer/Settings.tpl
+msgid "Are you sure you want to delete this Document Renderer?"
+msgstr "Sind Sie sicher, dass Sie diesen Dokument-Renderer löschen möchten?"
+
+#: templates/CRM/Civioffice/Form/DocumentRenderer/Settings.tpl
+#: templates/CRM/Civioffice/Form/LiveSnippet.tpl
+msgid "This action cannot be undone."
+msgstr "Diese Aktion kann nicht rückgängig gemacht werden."
+
+#: templates/CRM/Civioffice/Form/DocumentUpload.tpl
+msgid "Available Documents"
+msgstr "Verfügbare Dokumente"
+
+#: templates/CRM/Civioffice/Form/DocumentUpload.tpl
+msgid "Size"
+msgstr "Größe"
+
+#: templates/CRM/Civioffice/Form/DocumentUpload.tpl
+msgid "Actions"
+msgstr "Aktionen"
+
+#: templates/CRM/Civioffice/Form/DocumentUpload.tpl
+msgid "Delete File"
+msgstr "Datei löschen"
+
+#: templates/CRM/Civioffice/Form/DocumentUpload.tpl
+msgid "Download File"
+msgstr "Datei herunterladen"
+
+#: templates/CRM/Civioffice/Form/DocumentUpload.tpl
+msgid "No documents available"
+msgstr "Keine Dokumente verfügbar"
+
+#: templates/CRM/Civioffice/Form/DocumentUpload.tpl
+msgid "Upload More"
+msgstr "Weitere hochladen"
+
+#: templates/CRM/Civioffice/Form/Download.tpl
+msgid ""
+"Please click the download button only once. Clicking the button will trigger "
+"generating your zip file which may take a long time depending on the size "
+"and complexity of your documents.<br>Afterwards please always use the \"back "
+"to previous page\" button to finish this process"
+msgstr ""
+"Bitte klicken Sie den Knopf zum Herunterladen nur einmal. Das Klicken des "
+"Knopfes stößt das Erstellen einer ZIP-Datei an, was einige Zeit benötigen "
+"kann, abhängig von der Größe und der Komplexität Ihrer Dokumente. Klicken "
+"Sie danach immer den Knopf \"Zurück zu vorheriger Seite\", um den Prozess "
+"abzuschließen"
```

Comment:
Suggesting to replace *Klicken* with *Betätigen* (you know, touch devices can't "click"), although the source string doesn't provide a good wording in the first place.